pkg/private/tar/build_tar.py

```diff
@@ -273,31 +273,48 @@ def add_tree(self, tree_top, destpath, mode=None, ids=None, names=None):
           to_write[dest_dir + file] = content_path
 
     for path in sorted(to_write.keys()):
+      if file_attributes:
```

Comment:
Isn't this always true?

Reply:
yeah, at the moment; I copied how file_attributes was [used elsewhere in the script](https://github.com/bazelbuild/rules_pkg/blob/ed48f1c901a290e76815ede2da85133172d62218/pkg/private/tar/build_tar.py#L322-L325) because I didn't understand the context about why the script was already checking for it, and I figured it was safer to do it the same way.